src/Testcontainers.Spanner/.editorconfig

```diff
@@ -0,0 +1,241 @@
+root = true
```

Comment:
Could we please replace the configuration in the `.editorconfig` file with the configuration used by the other modules and stick to a plain code style (same as all the other modules)? Not all IDEs respect these files or configurations. While the indentation is nice, experience has shown that it does not work well in OSS projects.

Reply:
will do so